browser/react/index.js

```diff
@@ -47,7 +47,11 @@ ReactDOM.render(
     <MuiThemeProvider>
       <Router history={browserHistory}>
         <Route path="/" onEnter={onHomeEnter} >
-          <IndexRoute component={Login} />
+          <IndexRedirect to="/login" />
```

Comment:
it's been working. looks like '/home/login' doesn't work for me. i get this error in the console: "Warning: [react-router] Location "/home/login" did not match any routes". just '/home' renders the background and without the login or signup component so i made the indexredirect directly to login